tests/to_py.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::useless_vec)]
```

Comment:
Could you make a more localized fix, i.e. ideally directly for the let statement and nothing else?

Also, why couldn't you test this locally? The CI just uses the current stable version of Clippy.

Reply:
> Could you make a more localized fix, i.e. ideally directly for the let statement and nothing else?

Done

> Also, why couldn't you test this locally? The CI just uses the current stable version of Clippy.

I assumed it was using nightly. I just had to run `rustup update` for the warning to pop up.